### ISSUE TITLE
Water in the Nether

### ISSUE THREAD

**Issue by DarkObscurity** [BentoBoxWorld/BSkyBlock#240]
#### Is your feature request related to a problem?

Water is removed in the nether. (As supposed to in vanilla)

#### Describe the solution you'd like us to implement.

A setting to allow water placement in the nether.

#### Describe alternatives you've considered.

I haven't really, if I remembered correctly, this feature was once implemented, but now I just found out it doesn't work anymore and I can't find anything regarding it. (If it's supposed to work then this is a big and then I created the wrong request, i'm sorry if that's the case..)

**Comment by tastybento** [comment]
No, there's been no feature to enable water in the nether. I'm not sure it's really a core feature but it could be maybe.

**Comment by DarkObscurity** [comment]
Well it would be a really cool feature if toggle able, like this allows a server owner to give the nether world a little twist in terms of possibilities! 😄
